src/containers/donation/index.css

```diff
@@ -90,61 +158,238 @@
+  .tableBlueSection {
+    height: 3.5rem;
+  }
+}
+
+.leftActivityType,
+.rightActivityType {
   font-size: 21px;
   color: #dfdfdf;
+  display: inline-block;
+}
+
+@media only screen and (max-width: 530px) {
+  .leftActivityType,
+  .rightActivityType {
+    font-size: 16px;
+  }
+}
+
+@media only screen and (max-width: 420px) {
+  .leftActivityType,
+  .rightActivityType {
+    font-size: 14px;
+  }
 }
 
-.countNumbers {
+.leftCountNumbers,
+.rightCountNumbers {
   font-size: 2.8rem;
   color: white;
   padding: 0.5rem 0 0 0;
   display: block;
   line-height: 2.8rem;
 }
 
+@media only screen and (max-width: 650px) {
+  .leftCountNumbers,
+  .rightCountNumbers {
+    font-size: 2rem;
+    padding: 0.3rem 0 0 0;
+    line-height: 1.6rem;
+  }
+}
+
+@media only screen and (max-width: 420px) {
+  .leftCountNumbers,
+  .rightCountNumbers {
+    font-size: 1.6rem;
+    padding: 0.2rem 0 0 0;
+    line-height: 1.6rem;
+  }
+}
+
+.darkGreySection,
+.lightGreySection {
+  display: inline-block;
+  text-align: center;
+  width: 25%;
+  padding: 2rem 0 1rem;
+}
+
+@media only screen and (max-width: 1100px) {
+  .darkGreySection,
+  .lightGreySection {
+    width: 50%;
+    height: 20rem;
+    padding: 2.5%;
+  }
+}
+
+@media only screen and (max-width: 450px) {
+  .darkGreySection {
+    display: inline-block;
+    width: 100%;
+    height: 20rem;
+    padding: 2.5%;
+  }
+}
+
 .darkGreySection {
   background-color: #c4c4c4;
-  width: 25%;
-  height: 20rem;
-  padding: 2.3rem 0 2.3rem;
 }
 
 .lightGreySection {
   background-color: #f0f0f0;
-  width: 25%;
-  height: 20rem;
-  padding-top: 4%;
+}
+
+@media only screen and (max-width: 1100px) {
+  .lightGreySection,
+  .darkGreySection {
+    width: 50%;
+    height: 20rem;
+    padding-top: 7%;
+  }
 }
 
 .darkGreyColumn,
 .lightGreyColumn {
-  padding: 1.25rem 1.25rem;
-  font-size: 0.9rem;
+  margin-top: 10px;
+  padding: 1rem 1rem 0.8rem;
+  font-size: 1rem;
+}
+
+@media only screen and (max-width: 1270px) {
+  .darkGreyColumn,
+  .lightGreyColumn {
+    margin-top: 1.9rem;
+    padding: 1.1rem 1.1rem;
+    font-size: 0.9rem;
+    line-height: 1.25rem;
+  }
+}
+
+@media only screen and (max-width: 1100px) {
+  .darkGreyColumn,
+  .lightGreyColumn {
+    margin-top: 0.7rem;
+    padding: 1.1rem 1.1rem;
+    font-size: 1.1rem;
+    line-height: 1.5rem;
+  }
+}
+
+@media only screen and (max-width: 805px) {
+  .darkGreyColumn,
+  .lightGreyColumn {
+    margin-top: 0.7rem;
+    padding: 1.1rem 1.1rem;
+    font-size: 0.9rem;
+    line-height: 1.3rem;
+  }
+}
+
+@media only screen and (max-width: 650px) {
+  .darkGreyColumn,
+  .lightGreyColumn {
+    margin-top: 0.6rem;
+    padding: 1.1rem 1rem;
+    font-size: 0.8rem;
+    line-height: 1.2rem;
+  }
+}
+
+@media only screen and (max-width: 550px) {
+  .darkGreyColumn,
+  .lightGreyColumn {
+    margin-top: 0.5rem;
+    padding: 1.1rem 1rem;
+    font-size: 0.7rem;
+    line-height: 1.1rem;
+  }
+}
+
+@media only screen and (max-width: 450px) {
```

Comment:
usually we have 1, 2 max media queries. we don't do such specific media queries because it is hard to maintain (imagine oyu need to change something, now you need to change ALL these media queries).

in general what we do here is we use %s

please change the code to have at most 2 media queries, everywhere -- this css file is 420 lines which from my perspective is already a warning sign that there are some issues